src/modules/numberplate/carProvinceCodes.skip.ts

```diff
@@ -0,0 +1,141 @@
+import { createProvinceHashTable } from "./helpers";
+import { ProvinceDataset } from "./types.skip";
+
+export const carProvinceDataset: ProvinceDataset = [
```

Comment:
`carProvinceDataset` is an useless array which the result of its `hash-table` should be here and also please do the same for `bikeProvinceDataset` and `categories`. we have to decrease the size of our bundle.

I have to refactor my previous codes in some cases like this issue.

Reply:
can you elaborate, please?

Reply:
do you want me to write every single code manually?

Reply:
No, I meant there is no need to have an array of the `Provinces`, `Categories` and `Motorcycles` whereas we can create a dictionary of their mapped keys to values. Create their Dictionary and then replace the result with their list. 

Benefits:
1. There is no runtime hash-table creator.
2. Less bundle size.

Reply:
> No, I meant there is no need to have an array of the `Provinces`, `Categories` and `Motorcycles` whereas we can create a dictionary of their mapped keys to values. Create their Dictionary and then replace the result with their list.
> 
> Benefits:
> 
> 1. There is no runtime hash-table creator.
> 2. Less bundle size.

> No, I meant there is no need to have an array of the `Provinces`, `Categories` and `Motorcycles` whereas we can create a dictionary of their mapped keys to values. Create their Dictionary and then replace the result with their list.
> 
> Benefits:
> 
> 1. There is no runtime hash-table creator.
> 2. Less bundle size.

if i am not wrong we need something like this?
```js
const data = {
    Car: {
         10: "تهران",
         11: "تهران",
         12: "البرز",
         ...
    },
    Motorcycle: {
        700: "اصفحان"
        800: "اراک"
        ....
    },
    Category: {
        "دولتی": "ژ"
        "ص": "شخصی
        ....
    }
}
```
these two modules ( [getPlaceByIranNationalId](https://github.com/persian-tools/persian-tools/tree/master/src/modules/getPlaceByIranNationalId) &  [Sheba](https://github.com/persian-tools/persian-tools/tree/master/src/modules/sheba)) have similar APIs to numberplate module.


I used these to create the dataset.
can you please provide a simple example if you have time? 


because I am trying to understand the exact algorithm in your mind but I think I am not successful 😄

Reply:
kk got it